product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,352 @@
+
+If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing-pgd-for-edb-postgres-extended-server) and [Installing PGD for Community Postgresql](#installing-pgd-for-community-postgresql).
+
+## Worked example
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+For example, if we are installing EDB Postgres Advanced Server (epas) version 16, we would install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: 
+
+- `edb-pgd5-proxy` for PGD Proxy.
+- `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Ensure the database is initialized and started
+
+If it wasn't initialized and started by the database's package initialisation (or you are repeating the process), you will need to initialize and start the server.
+
+To see if the server is running, you can check the service. The service name for EDB Advanced Server is `edb-as-16` so run:
+
+```
+sudo systemctl status edb-as-16
+```
+
+If the server is not running, this will respond with:
+
+```
+○ edb-as-16.service - EDB Postgres Advanced Server 16
+     Loaded: loaded (/usr/lib/systemd/system/edb-as-16.service; disabled; preset: disabled)
+     Active: inactive (dead)
+```
+
+The "Active: inactive (dead)" tells us we will need to initialize and start the server.
+
+You will need to know the path to the setup script for your particular Postgres flavor.
+
+For EDB Postgres Advanced Server, this script can be found in `/usr/edb/as16/bin` as `edb-as-16-setup`. 
+This command needs to be run with the `initdb` parameter and we need to pass an option setting the database to use UTF-8.
+
+```
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+```
+
+Once the database is initialised, we will start it which will enable us to continue configuring the BDR extension.
+
+```
+sudo systemctl start edb-as-16
+```
+
+### Configure the BDR extension
+
+Installing EDB Postgres Advanced Server creates a system user `enterprisedb` with admin capabilities when connected to the database. We will be using this user to configure the BDR extension.
+
+#### Preload the BDR library
+
+We want the bdr library to be preloaded with other libraries. 
+EPAS has a number of libraries already preloaded, so we have to prefix the existing list with the BDR library.
+
+```
+echo -e "shared_preload_libraries = '\$libdir/bdr,\$libdir/dbms_pipe,\$libdir/edb_gen,\$libdir/dbms_aq'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
```

Comment:
Why are we using this cumbersome way to change settings, instead of using `ALTER SYSTEM` ?

Also, given that these instructions are intended for skilled users who opt out of the easier way, we can assume they can use an editor and that they are familiar with where the configuration files are and in which order they are processed, so we could just say "the following parameters must be set: ..."

Reply:
We did use ALTER SYSTEM and then I was told not to use it.

And this is a worked example. If you want the core instructions there are explicitly and cleanly llisted at the top of the page. 

And assuming a user can use an editor is a mistake; especially when they may be provisioning a system which doesn't have editors they are familiar with.